docs/api/guides/mocha.md

```diff
@@ -0,0 +1,21 @@
+# Using Enzyme with Mocha
+
+```bash
+npm i --save-dev enzyme-mocha
+```
+
+```jsx
+import { mount } from 'enzyme';
+import { describeWithDOM } from 'enzyme-mocha';
+
+describe('<Foo />', () => {
+
+  it('calls componentDidMount', () => {
+    spy(Foo.prototype, 'componentDidMount');
+    const wrapper = mount(<Foo />);
+    expect(Foo.prototype.componentDidMount.calledOnce).to.be.true;
```

Comment:
@ljharb it would be if we were using the sinon-chai plugin, but that would be assuming yet-another-library for this example, and should maybe be avoided?